app/scripts/views/generator-view.js

```diff
@@ -103,17 +123,22 @@ const GeneratorView = Backbone.View.extend({
 
     generate: function() {
         this.password = PasswordGenerator.generate(this.gen);
-        this.resultEl.text(this.password);
+        this.showPassword();
         const isLong = this.password.length > 32;
         this.resultEl.toggleClass('gen__result--long-pass', isLong);
     },
 
+    hideChange: function(e) {
+        this.hide = e.target.checked;
+        AppSettingsModel.instance.unset('generatorHide', { silent: true });
+        AppSettingsModel.instance.set('generatorHide', this.hide);
+        const label = this.$el.find('#gen__check-hide + label');
+        Tip.createTip(label[0], {title: this.hide ? Locale.genShowPass : Locale.genHidePass});
```

Comment:
Let's remove this. Once you have pressed the button, you clearly see the change in the UI.

Reply:
I'm not sure what you mean. This is tge callback called when you press tge button. There is a little bit of duplicate code in *templates/generator.hbs* to render tge initial state. We could remove the duplication by rerendering the template upon every model change, but I had a little trouble with this. I could investgate that more thoroughly if you think it's worth it.

Reply:
I see, I thought that was added to render a different tooltip that will be shown _after_ you have clicked the button. Nvm, ignore my comment then.